ajax/libs/elevator.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "elevator.js",
+  "version": "1.0.0",
+  "filename": "elevator.min.js",
+  "description": "Elevator.js fixes those awkward \"scroll to top\" moments the old fashioned way.",
+  "homepage": "http://tholman.com/elevator.js/",
+  "keywords": [
+    "scroll",
+    "top",
+    "elevator"
```

Comment:
It's good, we can add extra keywords let it better search.
But I still think we can add the same keyword from upstream.